src/main/java/com/patina/codebloom/common/db/repos/question/topic/service/QuestionTopicService.java

```
@@ -0,0 +1,33 @@
+package com.patina.codebloom.common.db.repos.question.topic.service;
+
+import java.util.Set;
+
+import org.springframework.stereotype.Service;
+
+import com.patina.codebloom.common.db.models.question.topic.LeetcodeTopicEnum;
+
+@Service
+public class QuestionTopicService {
+
+    public LeetcodeTopicEnum stringtoEnum(final String topic) {
+        try {
+            return LeetcodeTopicEnum.valueOf(topic);
+        } catch (IllegalArgumentException e) {
+            return LeetcodeTopicEnum.UNKNOWN;
+        }
+    }
```

if the enum doesn't exist, you can just throw an exception or let it bubble up and throw a ResponseStatusException saying that a topic was passed in that isn't supported.